renovate.json

```diff
@@ -34,29 +34,12 @@
       ],
       "datasourceTemplate": "github-tags",
       "extractVersionTemplate": "^v(?<version>.*)$"
-    },
-    {
-      "depNameTemplate": "defenseunicorns/zarf",
-      "fileMatch": ["README\\.md"],
-      "matchStrings": [
-        "- Zarf: (?<currentValue>.*)"
-      ],
-      "datasourceTemplate": "github-tags",
-      "extractVersionTemplate": "^v(?<version>.*)$"
-    },
-    {
-      "depNameTemplate": "defenseunicorns/zarf",
```

Comment:
just a thought on this. Do we add the settings.json in this repo for the schema validation and then add the match sting we use on that for uds-core for zarf?

I realize we sort of end up "dependent" on zarf version still in that case, but its just for that one scenario. Alternatively, uds could include the zarf schema in its repo and mitigate the need.

Reply:
🤔 currently we don't have the schema so not an issue in uds common tasks but its a good callout. Probably need to pull the zarf schema from uds-cli to handle if we want to drop dependency completely.

Maybe we could also add a task for `uds zarf prepare lint`, which should validate against the correct schema (noting that it doesn't actually handle the IDE feedback issue).

Reply:
Going to resolve this thread, think it makes sense to follow up once the above uds-cli issue is resolved.